1904-12-30.xml

```diff
@@ -0,0 +1,2634 @@
+                            yesterday and removed to the Deaconesses' Hospital previous to
+                            burial.</p>
+                    </div>
+                    <div type="item">
+                        <p>Cairo SYNDICATE, Ltd. - Registered 16th December. Capital £15,000, in £1
+                            shares. Objects : To adopt an agreement with E. C. Nicholls, and to
+                            carry on the business of concessionnaires, explorers, financiers,
+                            builders, contractors, railway and tramway proprietors, etc., in Egypt
+                            or elsewhere. No initial public issue.</p>
+                    </div>
+                    <div type="item">
+                        <p>Bank of Egypt.--The London Stock Exchange Committee have appointed
+                            Thursday next as a special settling day for Bank of Egypt. ---Further
+                            issue of 10,000 shares of £25 each (issued at £28), all paid (of which
+                            £12 10s is capital and £15 10s premium), Nos. 30,001 to 40,000. The
+                            Committee have also ordered the same to be quoted in the Official
+                            List.</p>
+                    </div>
+                    <div type="item">
+                        <p>Grand HOTEL, HELOUAN.-On the occasion of the Helouan races, we may state
+                            that special first-class return tickets, including lunch at the Grand
+                            Hotel Helouan, are issued at the Bab-el-Louk station, Cairo, at the
+                            offices of the Hotels Savoy, Angleterre, and Grand Continental, as well
+                            as at the Anglo-American Nile Steamer Company's offices and Carl
+                            Stangen's Reise Bureau, at the price of P.T. 20.</p>
+                    </div>
+                    <div type="item">
+                        <p>AUSTRALIA IN EGYPT.—Mr. John B. Gillison, F.I.A., F.F.A., manager for
+                            Great Britain and Ireland of the National Mutual Life Association of
+                            Australasia, Ltd., who has been staying at Shepheard's Hotel for some
+                            weeks past, is returning to England to-day by the Messageries Maritimes
+                            steamer from Alexandria. Mr. Gillison has been engaged in organising in
+                            Egypt a branch of the important life association which he represents.
+                            During next week an office will be opened in the conrt of the Cairo
+                            Stock Exchange under the control of Mr. S. de Bilinski..</p>
+                    </div>
+                </div>
+
+                <cb n="3"/>
+                <div type="section">
+                    <head>OUR NEW JUDGES.</head>
+                    <div type="item">
```

Comment:
no need to have this item within a section--just make the whole thing a single item with two headlines